common/changes/@itwin/physical-material-backend/update-deps_2022-01-25-17-52.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/physical-material-backend",
```

Comment:
I think you accidentally ran this without the branch specified. Running `rush change -b origin/release/3.0.x` should fix it.

There is an ongoing issue with our rush change python script that @skirby1996 is looking into that prevents any PRs from currently being completed.

Reply:
Yeah that was me...i'll fix.

Reply:
fixed.